README.md

```diff
@@ -13,6 +13,14 @@ _**Instant lightning-fast GraphQL API backed primarily by your PostgreSQL databa
 
 ## Documentation: [graphile.org/postgraphile](https://graphile.org/postgraphile)
 
+## Version 5 is coming: Open now to all sponsors
+
+April 2023 marked a big milestone for PostGraphile Version 5, when it entered the alpha stage. Work preparing for the final release is ongoing, but we now have a detailed migration guide, updated docs, an enhanced developer experience and much improved compatability!
+
+In our new blog post series, **we are beginning to lift the curtain of secrecy which has surrounded Version 5** for the past few years. Each week, Benjie will dive in to a key feature of this massive new release; Find the first article [on DEV.to](https://dev.to/graphile/intro-to-postgraphile-v5-part-1-replacing-the-foundations-3lh0).
+
+With beta just around the corner, now is the perfect time to join our sponsors and test V5. **If you don't try V5 until after its public release then it may be too late to comment on things that could have been improved.** If you already sponsor us, send us an email with your sponsor details and we'll send you access to [postgraphile.org](https://postgraphile.org/)
```

Comment:
```suggestion
Now is the perfect time to join our sponsors and get your feet wet with V5 - even if you're not yet ready to migrate to it fully. **If you don't try V5 until after its public release then it may be too late to comment on things that could have been improved.** If you already sponsor us, send us an email with your sponsor details and we'll send you access to [postgraphile.org](https://postgraphile.org/) and [grafast.org](https://grafast.org)!
```